index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Milestone1</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<div class="nav-back"></div>
+<body>
+    <!--home page header-->
+    <div class="header">
+        <a class="logo" href="index.html">
+            <h1>Angel Cooks</h1>
+        </a>
+        <nav>
+            <a class="nav-links" href="index.html">Home</a>
+            <a class="nav-links" href="about.html">About Me</a>
+        </nav>
+    </div>
+    <main>
+        <h2>Discover easy and tasty low FODMAP recipes!</h2>
+        <!--recipe cards-->
+        <div class="three-cards">
+            <div class="cards">
+                <img class="card-img" src="https://www.kimscravings.com/wp-content/uploads/2021/01/creamy-pasta-sauce-7.jpg">
+                <div class="card-body">
+                    <a href="recipe1.html"><button class="recipe-btn">Spicy Vodka Pasta</button></a>
```

Comment:
Not sure if a button is the best choice for these links. Since buttons are also clickable elements, it overrides the link so you can click around the title but not on the title itself. Maybe try replacing the button with a `<p>`, `<div>`, or another block level so you can keep your css formatting.